phpdocker/php-fpm/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM phpdockerio/php:8.3-fpm
+WORKDIR "/application"
+
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install \
+        php8.3-mysql \ 
+        php8.3-pgsql \ 
+        php8.3-sqlite3 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
```

```suggestion
    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*

ADD php-ini-overrides.ini /etc/php/8.3/fpm/conf.d/99-overrides.ini
```

The file can be added in the image, instead of using a volume.

---

I completely removed the custom php-ini because, as you mentioned it was kinda unecessary. The tests are still running fine without the extra configuration step so I omitted this aswell here.

Let me know what you think